esp/services/ws_workunits/ws_workunitsAuditLogs.cpp

```diff
@@ -1784,3 +1786,70 @@ bool CWsWorkunitsEx::onWUClusterJobSummaryXLS(IEspContext &context, IEspWUCluste
     }
     return true;
 }
+
+bool CWsWorkunitsEx::onWUGetThorJobQueue(IEspContext &context, IEspWUGetThorJobQueueRequest &req, IEspWUGetThorJobQueueResponse &resp)
+{
+    try
+    {
+        SecAccessFlags accessOwn, accessOthers;
+        getUserWuAccessFlags(context, accessOwn, accessOthers, true);
+
+        const char *cluster = req.getCluster();
+        const char *startDate = req.getStartDate();
+        const char *endDate = req.getEndDate();
+        unsigned maxJobQueueItemsToReturn = req.getMaxJobQueueItemsToReturn();
+        if (maxJobQueueItemsToReturn == 0)
+            maxJobQueueItemsToReturn = DefaultMaxJobQueueItemsInWUGetJobQueueResponse;
+
+        CDateTime fromTime, toTime;
+        StringBuffer fromstr, tostr;
```

Comment:
Should the time be set to 0:0:0 for fromstr, and 23:59:59 for tostr?  Otherwise it will only include part of the specified date

Reply:
Good point. I will check and, if needed, append 0:0:0 or 23:59:59.